docs/docsite/rst/porting_guides/porting_guide_2.5.rst

```diff
@@ -202,6 +202,12 @@ desired.
   * The :ref:`blockinfile module <blockinfile_module>` had its ``follow`` parameter removed because
     it inherently modifies the content of an existing file so it makes no sense to operate on the
     link itself.
+  * In Ansible-2.5.3, the :ref:`template module <template_module>` became more strict about its
+    ``src`` file being proper utf-8.  Previously, non-utf8 contents in a template module src file
+    would result in a mangled output file (the non-utf8 characters would be replaced with a unicode
+    replacement character).  Now, on Python2, the module will throw an error instead.  On Python3,
+    the module will first attempt to pass the non-utf8 characters through verbatim and fail if that
+    does not succeed.
```

Comment:
will the error message appear in python3 and python2, or only the latter?

Reply:
It should only appear in Pyhton2